templates/experiments/experiment_session_view.html

```diff
@@ -85,4 +89,28 @@ <h3 class="text-base font-semibold leading-7 mt-8 pl-4">Experiment details</h3>
   >
   </script>
   <script>SiteJS.tagMultiselect.setupTagSelects();</script>
+  <script>
+    document.addEventListener('DOMContentLoaded', function() {
```

Comment:
What is the purpose of this code? It seems like it could be done with an `htmx` `load` trigger.